openconcept/examples/B738_Dymos.py

```diff
@@ -168,7 +165,7 @@ def show_outputs(prob):
         x_label = 'Range (nmi)'
         y_labels = ['Altitude (ft)', 'Veas airspeed (knots)', 'Fuel used (lb)', 'Throttle setting', 'Vertical speed (ft/min)', 'Mach number', 'CL']
         phases = ['climb', 'cruise', 'descent','reserve_climb','reserve_cruise','reserve_descent','loiter']
-        oc.plot_trajectory(prob, x_var, x_unit, y_vars, y_units, phases,
+        plot_trajectory(prob, x_var, x_unit, y_vars, y_units, phases,
                         x_label=x_label, y_labels=y_labels, marker='-',
                         plot_title='737-800 Mission Profile')
 
```

Comment:
Should we remove this file? It is a duplicate of B738.py - I checked the diff between B738_Dymos.py and B738.py and two were identical.

Reply:
Good catch, I'll remove it